tests/test_init.py

```diff
@@ -18,6 +18,10 @@ def _module_is_loaded(module: str) -> bool:
     return os.system(f'{sys.executable} -c "{exe_str}"') != 0
 
 
+def test_failed():
+    pytest.fail('foo')
+
+
```

Comment:
Adding failed test in core to demonstrate this PR impact.
To be removed when the PR is ready.